src/fields/IntegerField.cpp

```diff
@@ -1,16 +1,16 @@
 #include "IntegerField.h"
 
-IntegerField::IntegerField(string name, int defaultValue, bool primary, bool unique, bool nullable, bool autoInc): 
-    Field(name, INTEGER, primary, unique, nullable, autoInc) { 
-    fieldValue = defaultValue;
-    autoIncrement = autoInc;
+IntegerField::IntegerField(string fieldName, int defaultValue, bool primaryKey, bool unique, bool nullable, bool autoIncrement): 
+    Field(fieldName, INTEGER, primaryKey, unique, nullable, autoIncrement) { 
+    _fieldValue = defaultValue;
+    _autoIncrement = autoIncrement; //Should this be passed to Field (above)?

```

Comment:
Yeah, the reason I say this is that for the integer field, autoincrement means that the field will be automatically filled. So if autoincrement is set, autofilled should be as well.